package.json

```diff
@@ -27,7 +27,9 @@
         "build": "vite build --config ./config/vite.config.js",
         "test": "npm run test-backend && npm run test-e2e",
         "test-with-build": "npm run build && npm test",
-        "test-backend": "cross-env TEST_BACKEND=1 node --test test/backend-test",
+        "test-backend": "node extra/test-backend.mjs",
+        "test-backend-node20": "cross-env TEST_BACKEND=1 node --test test/backend-test",
+        "test-backend-node22": "cross-env TEST_BACKEND=1 node --test test/backend-test/**/test-*.js",
```

Comment:
I suspected that the glob pattern is bugged, according to DigitalOcean's tool, it should match without sub-dir too, but in fact, it doesn't.

![image](https://github.com/user-attachments/assets/16c00976-2c2c-459e-af3d-f6f80dedf13b)